test/functional/test_framework/test_framework.py

```diff
@@ -191,6 +191,8 @@ def parse_args(self):
         parser.add_argument("--timeout-factor", dest="timeout_factor", type=float, help="adjust test timeouts by a factor. Setting it to 0 disables all timeouts")
         parser.add_argument("--v2transport", dest="v2transport", default=False, action="store_true",
                             help="use BIP324 v2 connections between all nodes by default")
+        parser.add_argument("--v1transport", dest="v1transport", default=False, action="store_true",
```

Comment:
nit, in the commit message of 547aacff08a2a5d786cb0fa6c7bae022ea651f8c `test: add -v1transport option and use it in test_runner`:
`s/-v1transport/--v1transport/`
`s/arei/are/`